actionpack/lib/action_dispatch/http/url.rb

```diff
@@ -249,12 +252,16 @@ def host
       end
 
       # Returns a \host:\port string for this request, such as "example.com" or
-      # "example.com:8080".
+      # "example.com:8080". Port is only included if it is not a default port
+      # (80 or 443)
       #
       #   class Request < Rack::Request
       #     include ActionDispatch::Http::URL
       #   end
       #
+      #   req = Request.new 'HTTP_HOST' => 'example.com'
```

Comment:
what does this new example give us, that the next one doesn't? they look the same?